README.md

```diff
@@ -102,51 +104,36 @@ Example command:
 
 Clone this repository to your desired folder:
 
-<!--
```

Comment:
# Your Project Design :red_circle:
## Users Index Page
![image](https://user-images.githubusercontent.com/93039359/205748044-f8b3a0bb-62b3-4d9b-8b5b-7cb85b9bc02d.png)
## Users Show Page
![image](https://user-images.githubusercontent.com/93039359/205748110-9400541c-93d2-42b6-9b07-50f42f8e916f.png)
## Posts Index Page
![image](https://user-images.githubusercontent.com/93039359/205748197-09c6a478-e86d-4b6d-960d-92b0234f956f.png)
## Post Show Page
![image](https://user-images.githubusercontent.com/93039359/205748293-a98cdab5-8b78-41ef-ae43-6af6e37d889b.png)

# Project Wireframe :green_circle:
## Users Index Page
![users](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAABYX6HCMRZA436OX62UYPSGUY4OOQGA)
## Users Show Page
![image](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAABYX6HCMENIL3WZ7CIWN65OGY4OOQHA)
## Posts Index Page
![image](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAABYX6HCNSW67KF2RPRRRHT2WY4OOQLA)
## Post Show Page
![image](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAABYX6HCMVSVTAXSQFII334WIY4OOQMA)

- [ ] Your styling is not very much similar to the wireframe. You don't need any special designing but you need to follow the wireframe :blush: Kindly make your design look similar to the wireframe :+1: